internal/check/valid_owner.go

```diff
@@ -180,8 +180,8 @@ func (v *ValidOwner) validateTeam(ctx context.Context, name string) *validateErr
 	org = strings.TrimPrefix(org, "@")
 	team := parts[1]
 
-	if org != v.orgName {
-		return newValidateError("Team %q does not belongs to %q organization.", team, v.orgName)
+	if !strings.EqualFold(org, v.orgName) {
```

Comment:
it's already done in line 176 (`name = strings.ToLower(name)`) but I need to double-check whether GitHub still does it as someone pointed out that it was changed: https://github.com/mszostok/codeowners-validator/pull/78#issuecomment-941445181